CRToast/CRToastConfig.m

```diff
@@ -742,6 +750,10 @@ - (BOOL)snapshotWindow {
     return (_options[kCRToastCaptureDefaultWindowKey] ? [_options[kCRToastCaptureDefaultWindowKey] boolValue] : kCRCaptureDefaultWindowDefault);
 }
 
+- (CGFloat)collisionTweak {
+    return _options[kCRToastCollisionTweakKey] ? [_options[kCRToastCollisionTweakKey] floatValue]: kCRToastCollisionTweakDefault;
```

Comment:
this is horribly nitpicky and I'm sorry ahead of time. I can't help myself as these things just stand out to me

but the spacing around the ` : ` doesn't match the others.  all overs have a leading and trailing space around the `:` of the ternary.